argostime/routes.py

```diff
@@ -166,6 +166,25 @@ def webshop_page(shop_id):
         show_variance=show_variance
         )
 
+@app.route("/stats")
+def statistics_page():
+
+    offers: List[ProductOffer] = ProductOffer.query.join(
+        Product).order_by(Product.name).all()
+
+    products: List[Product] = Product.query.all()
+
+    total_product: int = len(products)
```

Comment:
Lijkt me een beetje zonde om alle producten in te laden alleen om te kijken hoe lang de lijst is.

Kunnen we niet dit gebruiken: https://docs.sqlalchemy.org/en/20/orm/queryguide/query.html#sqlalchemy.orm.Query.count

Reply:
Hmm dat klinkt wel beter, maar misschien ook eens uitzoeken wat de SQLAlchemy v2.0 manier is om dat te doen

Reply:
Je hebt ook dit: https://docs.sqlalchemy.org/en/20/core/functions.html#sqlalchemy.sql.functions.count

Reply:
dat lijkt me de nieuwe versie inderdaad